build/root/WORKSPACE

```diff
@@ -63,6 +63,14 @@ docker_pull(
 )
 
 docker_pull(
+    name = "debian-hyperkube-base-amd64",
```

Comment:
we don't actually need this change, since release-1.8 doesn't build the hyperkube image with bazel.

Reply:
I removed this change.